aws/Dockerfile

```diff
@@ -12,4 +12,7 @@ RUN echo "Install AWS" && \
     echo "Install CI Helper" && \
     curl -sSL https://github.com/rande/gitlab-ci-helper/releases/download/${CI_HELPER_VERSION}/alpine-amd64-gitlab-ci-helper -o /usr/bin/ci-helper && \
     chmod 755 /usr/bin/ci-helper && \
-    echo "Done install CI Helper"
+    echo "Done install CI Helper" && \
+
+    echo "Adding an up to date mime-types definition file" && \
+    curl -sSL https://salsa.debian.org/debian/mime-support/raw/master/mime.types -o /etc/mime.types && \
```

Comment:
Alpine here, not a Debian

Reply:
Sure, but still, there is a problem as all the mime types are not known...

Even in Alpine, the main definition file is `/etc/mime.types` and its format is not related to Debian.

```
/data # python
Python 2.7.14 (default, Feb 22 2018, 22:26:54)
[GCC 5.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import mimetypes
>>> mimetypes.guess_type('aze.woff2')
(None, None)
>>> mimetypes.guess_type('aze.woff')
(None, None)
>>> mimetypes.guess_type('aze.txt')
('text/plain', None)
>>> mimetypes.knownfiles
['/etc/mime.types', '/etc/httpd/mime.types', '/etc/httpd/conf/mime.types', '/etc/apache/mime.types', '/etc/apache2/mime.types', '/usr/local/etc/httpd/conf/mime.types', '/usr/local/lib/netscape/mime.types', '/usr/local/etc/httpd/conf/mime.types', '/usr/local/etc/mime.types']
```

What do you think about it @mremi?